app/forms.py

```diff
@@ -0,0 +1,14 @@
+from flask_wtf import FlaskForm
+from wtforms import StringField, PasswordField, BooleanField, SubmitField
+from wtforms.validators import DataRequired
+
+class LoginForm(FlaskForm):
+    username = StringField('Username', validators=[DataRequired()])
+    password = PasswordField('Password', validators=[DataRequired()])
+    remember_me = BooleanField('Remember Me')
+    submit1 = SubmitField('Log In')
+
+class SignInForm(FlaskForm):
+    username = StringField('UsernameSignUp', validators=[DataRequired()])
+    password = PasswordField('PasswordSignUp', validators=[DataRequired()])
```

Comment:
These field names show up really weird on the the web-page. These could probably be changed to 'Username' and 'Password' respectively